<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-PINDU21" failures="1" tests="7" name="Default test" time="56.67" errors="0" timestamp="2023-05-29T21:55:45 IST">
  <testcase classname="com.tutorialsninja.qa.testcases.Search" name="verifySearchwithValidProduct" time="1.139"/>
  <testcase classname="com.tutorialsninja.qa.testcases.Search" name="verifySearchwithInvalidProduct" time="1.13">
    <failure type="java.lang.AssertionError" message="expected [There is noo product that matches the search criteria.] but found [There is no product that matches the search criteria.]">
      <![CDATA[java.lang.AssertionError: expected [There is noo product that matches the search criteria.] but found [There is no product that matches the search criteria.]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.tutorialsninja.qa.testcases.Search.verifySearchwithInvalidProduct(Search.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifySearchwithInvalidProduct -->
  <testcase classname="com.tutorialsninja.qa.testcases.Register" name="verifyRegiterWithMandatoryFields" time="2.035"/>
  <testcase classname="com.tutorialsninja.qa.testcases.Register" name="verifyRegiterWithAllFields" time="2.03"/>
  <testcase classname="com.tutorialsninja.qa.testcases.Login" name="verifyLoginWithValidCredentials" time="1.115"/>
  <testcase classname="com.tutorialsninja.qa.testcases.Login" name="verifyLoginWithValidCredentials" time="1.204"/>
  <testcase classname="com.tutorialsninja.qa.testcases.Login" name="verifyLoginWithInvalidCredentials" time="0.978"/>
</testsuite> <!-- Default test -->
